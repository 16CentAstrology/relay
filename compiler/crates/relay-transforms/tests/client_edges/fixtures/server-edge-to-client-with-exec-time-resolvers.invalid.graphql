#expected-to-throw
query TestQuery @exec_time_resolvers {
  me {
    client_recommendation {
      id
    }
  }
}

fragment ClientRecommendationResolverFragment on User {
  __typename
}

# %extensions%

type ClientOnlyType implements Node {
  id: ID!
}

extend type User {
  client_recommendation: ClientOnlyType @relay_resolver(fragment_name: "ClientRecommendationResolverFragment", import_path: "ClientRecommendationResolver")
}
