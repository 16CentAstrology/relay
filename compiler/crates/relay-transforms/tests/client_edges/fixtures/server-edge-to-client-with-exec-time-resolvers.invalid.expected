==================================== INPUT ====================================
#expected-to-throw
query TestQuery @exec_time_resolvers {
  me {
    client_recommendation {
      id
    }
  }
}

fragment ClientRecommendationResolverFragment on User {
  __typename
}

# %extensions%

type ClientOnlyType implements Node {
  id: ID!
}

extend type User {
  client_recommendation: ClientOnlyType @relay_resolver(fragment_name: "ClientRecommendationResolverFragment", import_path: "ClientRecommendationResolver")
}
==================================== ERROR ====================================
✖︎ Server to client edges are not supported in exec time resolvers. Please consider disable exec time resolver on the query for now, or not using server to client edges.

  server-edge-to-client-with-exec-time-resolvers.invalid.graphql:4:5
    3 │   me {
    4 │     client_recommendation {
      │     ^^^^^^^^^^^^^^^^^^^^^
    5 │       id
